packages/core/src/experimental/myAccountSeverSideProps.ts

```diff
@@ -0,0 +1,42 @@
+import type { Locator } from '@vtex/client-cms'
+import type { GetServerSideProps } from 'next'
+
+import {
+  type GlobalSectionsData,
+  getGlobalSectionsData,
+} from 'src/components/cms/GlobalSections'
+
+import { injectGlobalSections } from 'src/server/cms/global'
+
+export type MyAccountProps = {
+  globalSections: GlobalSectionsData
+}
+
+export const getServerSideProps: GetServerSideProps<
```

Comment:
@eduardoformiga we should create the getServerSideProps for each page in my-account. To do it:

- We need to create a function to get the global sections
- We need to create a function to handle authorization and permission.